src/ComponentBase.ts

```diff
@@ -41,7 +41,7 @@ interface StoreSubscriptionInternal<P, S> extends StoreSubscription<P, S> {
     _subscriptionKey?: string;
 }
 
-export abstract class ComponentBase<P extends React.Props<any>, S extends Object> extends React.Component<P, S> {
+export abstract class ComponentBase<P extends React.Props<any>, S extends Object> extends React.PureComponent<P, S> {
```

Comment:
Why make this change? `ComponentBase` overrides `shouldComponentUpdate` so there is no benefit in using `PureComponent`. Actually, it is misleading since the reader would expect shallow comparison but the default `shouldComponentUpdateComparator` is not doing that.

Reply:
unintentional